docs/dataproc.md

```diff
@@ -0,0 +1,99 @@
+## Integration with DataProc / PySpark
+
+Bigflow provides integration with Pyspark running on Dataproc.
+It allows to adapt any python callable/function with PySpark code into `dataflow` Job.
+
+### Installation
+
+```shell
+pip install bigflow[dataproc]
+```
+
+### Define basic job
+
+Each PySpark job must have an entry point: python callable, passed as `driver` parameter. 
+Also some additional deployment options must be specified: GCP project name, GCP region name, GCS bucket/path to store deployments artifacts.
+
+```python
+import bigflow
+import bigflow.dataproc
+
+import operator
+import pyspark
+
+def do_pyspark_pipeline(context: bigflow.JobContext):
+    sc = pyspark.SparkContext()
+    # Any PySpark code....
+    print(sc.range(100).reduce(operator.add))
+
+pyspark_job = bigflow.dataproc.PySparkJob(
+    id='pyspark_job',
+    driver=do_pyspark_pipeline,
+    staging_location="gs://bucket-name/path",
+    gcp_project_id="my-project",
+    gcp_region="us-west-1",
+    # ... other pyspark job options
+)
+```
+
+Value of `driver` arguments must be `pickle`able function: it may be global function, an object with `__call__` method,
```

Comment:
Value of `driver` arguments -> Value of a `driver` argument
?

Reply:
done